README.md

```diff
@@ -65,7 +65,7 @@ expect(() => cat.lives, throwsRangeError);
 
 // We can calculate a response at call time.
 var responses = ["Purr", "Meow"];
-when(cat.sound()).thenAnswer(() => responses.removeAt(0));
+when(cat.sound()).thenAnswer((Invocation invocation) => responses.removeAt(0));
```

Comment:
```suggestion
when(cat.sound()).thenAnswer((_) => responses.removeAt(0));
```

The dart idiom for an argument necessary to fit a signature, but unused, is to name it `_`.

We also don't need the type annotation on `Invocation` because it can be inferred by the signature of the argument to `.thenAnswer`.